src/Administration/Resources/app/administration/src/app/snippet/de-DE.json

```diff
@@ -754,7 +754,9 @@
   },
   "sw-property-search": {
     "placeholderSearch": "Suche nach Eigenschaften ...",
-    "selected": "{count} Option ausgewählt | {count} Optionen ausgewählt"
+    "selected": "{count} Option ausgewählt | {count} Optionen ausgewählt",
+    "noPropertiesFound": "Keine Optionen gefunden",
+    "noPropertiesFoundDescription": "versuchen Sie es mit einem anderen Sucher"
```

Comment:
```suggestion
    "noPropertiesFoundDescription": "Versuchen Sie es mit einem anderen Sucher"
```

Reply:
ping @GitEvil

Reply:
This is supposed to be a notification text, right? If thats the case, we should go without the title (its not needed anymore) and incorporate all information into the message. I adjusted @PaddyS 's suggestion already. But be sure to get rid of the title in the notificationCreate and the snippet.

Reply:
@GitEvil I changed my commit, but it is not supposed to be a GrowlNotification. Not sure if that's what you mean? 

You can see the empty state I added in the screenshots in the issue description.